crates/astria-sequencer/src/fees/storage/keys.rs

```diff
@@ -71,28 +66,21 @@ mod tests {
     fn keys_should_not_change() {
         // NOTE: This helper struct is just to avoid having 14 snapshot files to contend with.
         // NOTE: `BLOCK` is only used in the ephemeral store, so isn't included here.
-        struct Helper;
-        impl Display for Helper {
-            fn fmt(&self, formatter: &mut Formatter<'_>) -> fmt::Result {
-                writeln!(formatter, "{BRIDGE_LOCK}")?;
-                writeln!(formatter, "{BRIDGE_SUDO_CHANGE}")?;
-                writeln!(formatter, "{BRIDGE_UNLOCK}")?;
-                writeln!(formatter, "{FEE_ASSET_CHANGE}")?;
-                writeln!(formatter, "{FEE_ASSET_PREFIX}")?;
-                writeln!(formatter, "{FEE_CHANGE}")?;
-                writeln!(formatter, "{IBC_RELAY}")?;
-                writeln!(formatter, "{IBC_RELAYER_CHANGE}")?;
-                writeln!(formatter, "{IBC_SUDO_CHANGE}")?;
-                writeln!(formatter, "{ICS20_WITHDRAWAL}")?;
-                writeln!(formatter, "{INIT_BRIDGE_ACCOUNT}")?;
-                writeln!(formatter, "{SEQUENCE}")?;
-                writeln!(formatter, "{SUDO_ADDRESS_CHANGE}")?;
-                writeln!(formatter, "{TRANSFER}")?;
-                writeln!(formatter, "{VALIDATOR_UPDATE}")?;
-                Ok(())
-            }
-        }
-        insta::assert_snapshot!(Helper);
+        assert_snapshot!("bridge_lock_fees_key", BRIDGE_LOCK);
```

Comment:
all of these are so nice (looking at the names of the snapshots in your next commit)

Reply:
😁